boot/bootutil/src/loader.c

```diff
@@ -283,6 +283,10 @@ static int
 boot_version_cmp(const struct image_version *ver1,
                  const struct image_version *ver2)
 {
+    BOOT_LOG_DBG("boot_version_cmp: ver1 %d.%d.%d vs ver2 %d.%d.%d",
+                 ver1->iv_major, ver1->iv_minor, ver1->iv_revision,
+                 ver2->iv_major, ver2->iv_minor, ver2->iv_revision);
+
```

Comment:
this should just log the full versions part if the Kconfig is enabled, having it output in 2 parts is ugly